source/Government.cpp

```diff
@@ -137,6 +137,8 @@ void Government::Load(const DataNode &node)
 		{
 			if(key == "provoked on scan")
 				provokedOnScan = false;
+			else if(key == "restricted")
```

Comment:
I feel like the keyword could be clearer, and the LocationFilter variable name is pretty clear.
```suggestion
			else if(key == "travel restrictions")
```